app/servers.json

```diff
@@ -0,0 +1,21 @@
+[{
+  "name": "Community Demo",
+  "etl": "/etl-server/etl/",
+  "amrs": "/amrs/ws/rest/v1/"
+}, {
+  "name": "Production",
+  "etl": "https://amrsreporting.ampath.or.ke:8002/etl/",
+  "amrs": "https://amrs.ampath.or.ke:8443/amrs/ws/rest/v1/"
+}, {
+  "name": "Production Demo",
+  "etl": "https://amrsreporting.ampath.or.ke:8003/etl/",
+  "amrs": "https://amrs.ampath.or.ke:8443/amrs/ws/rest/v1/"
+}, {
+  "name": "Test",
+  "etl": "https://test1.ampath.or.ke:8002/etl/",
```

Comment:
Move production to the top of the list as the users are used to this being the first item. Make production demo the second item.